Sources/Interaction/Manipulators/MouseCameraTrackballZoomManipulator/index.js

```diff
@@ -74,7 +74,7 @@ function vtkMouseCameraTrackballZoomManipulator(publicAPI, model) {
 
     const camera = renderer.getActiveCamera();
 
-    const dyf = delta;
+    const dyf = 1 - delta / 10;
```

Comment:
can we use a more descriptive name for `dyf` here? And add a comment that explains why `10`?

Reply:
So I'm trying to guess what `dyf` is... maybe `deltaYFactor`? :thinking: 

`delta` used to be the scroll value given from the mouse wheel event, and it was pretty arbitrary: https://github.com/Kitware/vtk-js/blob/051f85534bb6053f17f8fffb5f7f6c97e121eb60/Sources/Rendering/Core/RenderWindowInteractor/index.js#L410-L419

`1 - delta / 10` is just as arbitrary, but it is done so that the "speed" of zooming during the scroll stays consistent now that we track the scroll value differently (see below):
https://github.com/Kitware/vtk-js/blob/2b68fe93b400afd6737f985a1879c564040fc59d/Sources/Rendering/Core/RenderWindowInteractor/index.js#L425

Reply:
There's a comment in there somewhere. ;) You can say `const scale = 10; // to keep delta Y around 1`.